src/js/fluxcontroller.js

```diff
@@ -958,6 +967,23 @@ define(function (require, exports, module) {
     };
 
     /**
+     * Given a webpack context of stores, returns all Flux initialized stores
+     *
+     * @param {object} context all the store files in a context
+     *
+     * @return {object} An object of initialized stores
```

Comment:
It would be nice to provide stronger type annotations here. In particular, when objects are used as maps they can and should be annotated like `Object.<T1, T2>` (note the capital O).